source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -230,6 +231,29 @@ thumbnail.
 ~~~
 
 
+#### Additional Image Link
+
+Χρησιμοποιείται για επιπρόσθετες φωτογραφίες του προϊόντος. Επιπλέον
+φωτογραφίες του ίδιου προϊόντος από διαφορετικές οπτικές γωνίες ή
+με διαφορετικό τρόπο παρουσίασης ή συνδυασμοί των παραπάνω ενισχύουν
+την παρουσίαση του προϊόντος στο <%= settings.site_name.capitalize %>.
+
+Το πεδίο είναι προαιρετικό αλλά συνίσταται η χρήση του. Μπορεί να εμφανίζεται
+πολλές φορές σε ένα feed, μιά φορά για κάθε διαφορετική φωτογραφία.
+Ισχύουν οι ίδιοι περιορισμοί με το [Image Link](#image-link)
+
+##### Παράδειγμα
+
+~~~ xml
+<image>http://www.mydomain.com/photos/1.jpg</image>
+<imageurl>http://www.mydomain.com/photos/b&amp;amp;b.jpg</imageurl>
```

Comment:
Are you sure you want this URL to be displayed as `http://www.mydomain.com/photos/b&amp;amp;b.jpg`?